lib/cli/node-modules-path.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var path  = require('path');
+
+module.exports = function(context) {
+  var nodePath = process.env.EMBER_NODE_PATH;
+  var contextPath = path.resolve(context);
+
+  if (nodePath) {
+    var nodeModulesPath = path.resolve(nodePath);
+
+    if (contextPath.indexOf(nodeModulesPath) === 0) {
+      return path.resolve(contextPath,'node_modules');
```

Comment:
Right, we discussed. The more specific path is correct, as it may be a specific subdirectory/nested dep in `node_modules/`